Error compiling for armv7-apple-ios

This is 607f74df2ad6b6fb32ce5b862a9e48a8c4afdad2.

~~~
rustc: x86_64-apple-darwin/stage2/lib/rustlib/armv7-apple-ios/lib/libstd
warning: dropping unsupported crate type `dylib` for target `armv7-apple-ios`
../src/libstd/rand/os.rs:215:43: 215:52 error: found zero-size struct in foreign module, consider adding a member to this struct, #[deny(improper_ctypes)] on by default
../src/libstd/rand/os.rs:215         fn SecRandomCopyBytes(rnd: *const SecRandom,
                                                                       ^~~~~~~~~
error: aborting due to previous error
make: *** [x86_64-apple-darwin/stage2/lib/rustlib/armv7-apple-ios/lib/stamp.std] Error 101
[2] sirver@raph(git)-[master] > mvim ../src/libstd/rand/os.rs
~~~

--- a/src/libstd/rand/os.rs
+++ b/src/libstd/rand/os.rs
@@ -185,7 +185,7 @@ mod imp {
     use io;
     use mem;
     use rand::Rng;
-    use libc::{c_int, size_t};
+    use libc::{c_int, c_void, size_t};
 
     /// A random number generator that retrieves randomness straight from
     /// the operating system. Platform sources:
@@ -203,8 +203,9 @@ mod imp {
         _dummy: (),
     }
 
-    #[repr(C)]
-    struct SecRandom;
+    // Fake definition; this is actually a struct, but we don't use the
+    // contents here.
+    type SecRandom = c_void;
 
     #[allow(non_upper_case_globals)]
     const kSecRandomDefault: *const SecRandom = 0 as *const SecRandom;

